src/main/java/org/wso2/diagnostics/watchers/logwatcher/LogWatcher.java

```diff
@@ -24,12 +24,28 @@
 import java.io.IOException;
 import java.io.RandomAccessFile;
 
+import java.util.LinkedList;
+import java.util.regex.Pattern;
+
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 
 public class LogWatcher extends Thread {
 
     private static final Logger log = LogManager.getLogger(LogWatcher.class);
+    private static final int MAX_CONTEXT_SIZE = 5;
```

Comment:
Shall we make these values configurable from the config.toml of the diagnostic tool?